.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
Is this accurate? #56 addresses ES6 usage to preserve backward compatibility. @albertinad